projects/shapeshift/index.js

```diff
@@ -0,0 +1,47 @@
+const sdk = require("@defillama/sdk");
+const { sumTokensAndLPsSharedOwners } = require("../helper/unwrapLPs");
+
+// Contracts
+const StakingYieldContract = "0xDd80E21669A664Bce83E3AD9a0d74f8Dad5D9E72";
+
+// Tokens Or LPs
+const ETH_FOX_UNIV2 = "0x470e8de2eBaef52014A47Cb5E6aF86884947F08c";
+const FOX = "0xc770eefad204b5180df6a14ee197d99d808ee52d";
+
+const calcTvl = async (tokenOrLp, bool) => {
+  const balances = {};
+  let chainBlocks = {};
+
+  await sumTokensAndLPsSharedOwners(
+    balances,
+    [[tokenOrLp, bool]],
+    [StakingYieldContract],
+    chainBlocks["ethereum"],
+    "ethereum",
+    (addr) => addr
+  );
+
+  return balances;
+};
+
+/*** Staking of native token (FOX) TVL portion ***/
+const staking = async () => {
+  return calcTvl(FOX, false);
+};
+
+const ethTvl = async () => {
+  return calcTvl(ETH_FOX_UNIV2, true);
+};
+
+module.exports = {
```

Comment:
for this adapter I'd just `staking` and `pool2`, imo it will drive the LOC count down by a lot

Reply:
yeah, do you want me try to change it?